server/models/contractor.js

```diff
@@ -0,0 +1,31 @@
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
```

Comment:
to use mongoose.schema and mongoose.model need to import mongoose package
```suggestion
const {Schema, model} = require('mongoose');
```
can destructure to just use Schema and mode directly